plugins/slicer/MONAILabel/MONAILabel.py

```diff
@@ -479,10 +479,26 @@ def updateGUIFromParameterNode(self, caller=None, event=None):
 
         if self.models and [k for k, v in self.models.items() if v["type"] == "segmentation"]:
             self.ui.segmentationCollapsibleButton.collapsed = False
+            self.ui.segmentationCollapsibleButton.show()
+        else:
+            self.ui.segmentationCollapsibleButton.hide()
+
         if self.models and [k for k, v in self.models.items() if v["type"] in ("deepgrow", "deepedit")]:
             self.ui.deepgrowCollapsibleButton.collapsed = False
+            self.ui.deepgrowCollapsibleButton.show()
+        else:
+            self.ui.deepgrowCollapsibleButton.hide()
+
         if self.models and [k for k, v in self.models.items() if v["type"] == "scribbles"]:
             self.ui.scribblesCollapsibleButton.collapsed = False
+            self.ui.scribblesCollapsibleButton.show()
+        else:
+            self.ui.scribblesCollapsibleButton.hide()
+
+        if self.info.get("trainers", {}):
+            self.ui.aclCollapsibleButton.show()
+        else:
+            self.ui.aclCollapsibleButton.hide()
```

Comment:
@SachidanandAlle, we are running two segmentation apps that do not make use of any trainers.

With these code changes our workflow of using MONAILabel to segment images from scratch or to use pre-segmentations does not work any more because the buttons 'Next Sample' and 'Submit Sample' are not available any more.

I can understand the rationale behind these code changes but for our use case it doesn't work. Is it possible to discuss these code changes? Thanks.